dali/test/python/test_operator_decoders_image.py

```diff
@@ -90,26 +93,159 @@ def log(img_type, size, device, threads):
+                               shard_id = 0,
+                               num_shards = 1,
+                               name = "Reader")
+
+    # ToDo make window random?
+    images_crop_1 = fn.decoders.image_crop(jpegs,
+                                           device = device,
+                                           output_type = types.RGB,
+                                           hw_decoder_load = 0.7,
+                                           crop = (224, 224),
+                                           crop_pos_x = 0.3,
+                                           crop_pos_y = 0.2)
+
+    images = fn.decoders.image(jpegs,
+                               device = device,
+                               hw_decoder_load = 0.7,
+                               output_type = types.RGB)
+    # ToDo make window random?
+    images_crop_2 = fn.crop(images,
+                            crop = (224, 224),
+                            crop_pos_x = 0.3,
+                            crop_pos_y = 0.2)
+
+    return images_crop_1, images_crop_2
+
+def run_decode_crop(path, img_type, batch, device, threads):
+    data_path = os.path.join(test_data_root, path, img_type)
+    pipe = create_decoder_crop_pipeline(data_path=data_path, batch_size=batch, num_threads=threads, device_id=0, device=device, prefetch_queue_depth=1)
+    pipe.build()
+    iters = math.ceil(pipe.epoch_size("Reader") / batch)
+    for _ in range(iters):
+        out_1, out_2 = pipe.run()
+        for img_1, img_2 in zip(out_1, out_2):
+            img_1 = to_array(img_1)
+            img_2 = to_array(img_2)
+            assert np.allclose(img_1, img_2)
+
+def test_image_decoder_crop():
+    for device in {'cpu', 'mixed'}:
+        for threads in {4}:
+            for size in {10}:
+                for img_type in test_good_path:
+                    yield run_decode_crop, good_path, img_type, size, device, threads
+
+
+@pipeline_def
+def create_decoder_random_crop_pipeline(data_path, device):
+    seed = 1234
+    jpegs, _ = fn.readers.file(file_root = data_path,
+                               shard_id = 0,
+                               num_shards = 1,
+                               name = "Reader")
+
+    images_random_crop_1 = fn.decoders.image_random_crop(jpegs,
+                                                         device = device,
+                                                         output_type = types.RGB,
+                                                         hw_decoder_load = 0.7,
+                                                         seed = seed)
+    images_random_crop_1 = fn.resize(images_random_crop_1, size = (224,224))
+
+    images = fn.decoders.image(jpegs,
+                               device = device,
+                               hw_decoder_load = 0.7,
+                               output_type = types.RGB)
+    images_random_crop_2 = fn.random_resized_crop(images, size = (224,224), seed = seed)
```

Comment:
why do we compare with random_resized_crop and not with plain random_crop?

Reply:
Because we don't have a standalone `random_crop`.